components/prism-cpp.js

```diff
@@ -17,19 +17,16 @@
 			// This will capture the class name before destructors like:
 			//   Foo::~Foo() {}
 			/\b[A-Z_]\w*(?=\s*::\s*~\w+\s*\()/i,
-			{
-				// This also intends to capture the class name of method implementations but here the class has template
-				// parameters, so it can't be a namespace (until C++ adds generic namespaces).
-				pattern: /\w+(?=\s*<(?:[^<>]|<(?:[^<>]|<[^<>]*>)*>)*>\s*::\s*\w+\s*\()/,
-				inside: null // see below
```

Comment:
Do we know what this referred to before?

Reply:
Nothing. I think it used to refer to something during the development of #2348, that changed, and I forgot to remove it.